pkg/asset/cluster/tfvars.go

```diff
@@ -668,6 +669,12 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 			imageCacheIP = installConfig.Config.Platform.BareMetal.BootstrapProvisioningIP
 		}
 
+		machineCIDR := &installConfig.Config.Networking.MachineNetwork[0].CIDR
+		virtHostIP, err := cidr.Host(&machineCIDR.IPNet, 1)
```

Comment:
I wouldn't expect this to work in general. I think we need to find an IP address of the local host that is in the machine network. There's no guarantee that this will be the first IP address in that CIDR. it could be anything.

Reply:
Do you have any ideas how to detect that? I could not find anything in terraform...